style.css

```diff
@@ -635,300 +622,231 @@ input {
     line-height: 48px;
   }
 
-  .main__shop a {
+  .shop__link {
     position: absolute;
     color: white;
     top: 310px;
     left: 45px;
     font-size: 20px;
     line-height: 19px;
-    /*width: 100%;*/
   }
 
-  .main__shop a::after {
+  .shop__link::after {
     content: "\2192";
-    display: inline-block; 
+    display: inline-block;
     font-size: 20px;
     margin-left: 5px;
     width: 20px;
   }
 
-  .main h3{
+  .main h3 {
     font-size: 30px;
     line-height: 29px;
-    letter-spacing: 0.5px; 
+    letter-spacing: 0.5px;
     margin-bottom: 20px;
   }
-  
-  .main__product {
+
+  .product {
     margin-bottom: 80px;
   }
-  
-  .main__product_item:nth-child(n+4) {
+
+  .product__content:nth-child(n+4) {
     display: none;
     max-width: none;
-  } 
-  
-  .main__product_line {
+  }
+
+  .product__line {
     display: none;
     max-width: none;
   }
   
-  .main__product_item {
+  .product__content {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".product__content" to come before selector ".product__content:nth-child(n+2)" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Mandri_Firsttask/pullRequest?prid=5484031)